src/components/events/partials/modals/EventDetails.tsx

```diff
@@ -164,7 +164,7 @@ const EventDetails = ({
 			accessRole: "ROLE_UI_EVENTS_DETAILS_COMMENTS_VIEW",
 			name: "tobira",
 			page: EventDetailsPage.Tobira,
-			hidden: tobiraError?.message?.includes("503"),
+			hidden: (tobiraError === undefined || tobiraError?.message === undefined || tobiraError?.message?.includes("503")),
```

Comment:
This also hides the Tobira tab when it is configured correctly in Opencast. Maybe you can add 
```
const tobiraStatus = useAppSelector(state => getEventDetailsTobiraStatus(state));
```
in line 103 and then here use
```suggestion
			hidden: ((tobiraError === undefined || tobiraError?.message === undefined) && tobiraStatus === "failed")
				|| !!tobiraError?.message?.includes("503"),
```

and then the same for `SeriesDetails` and `NewSeriesWizard`.
The `!!` part of `!!tobiraError?.message?.includes("503")` should only be necessary in `NewSeriesWizard`.

Reply:
Thanks for the review. I forgot that we only assume Tobira to be unavaible if we get a 503 response explicitly. The new changes hopefully offer a better solution.